### PR TITLE
Add unit tests

### PR DESCRIPTION
**Pull request by IrinaSendetskaya**

### CODE REVIEW

File: src/app/components/add-course/add-course.component.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
+<p>
+  New Course
+</p>
+<div class="new-course-form" method="POST">
```
Comment (nastabulak) [inline src/app/components/add-course/add-course.component.html]: new-course__form

File: src/app/components/add-course/add-course.component.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  <div class="new-course-form-group">
```
Comment (nastabulak) [inline src/app/components/add-course/add-course.component.html]: new-course__form-group

File: src/app/components/add-course/add-course.component.html [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+      [(authorsConfirmList)]="courseInput.authors"
+      #authors
+      class="new-course-form-control"
+      required
+    ></app-drag-drop>
+  </div>
+  <div class="new-course-form-group">
+    <button
+      [disabled]="nameCourse.invalid || description.invalid || duration.invalid"
+      class="button button_submit"
```
Comment (nastabulak) [inline src/app/components/add-course/add-course.component.html]: button - element submit - modificator new-course - block

File: src/app/components/courses/courses.component.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
+<p>
+  Courses catalog
+</p>
+
+<div class="content-courses__list" method="GET">
```
Comment (nastabulak) [inline src/app/components/courses/courses.component.html]: content? is it block? or element?

File: src/app/components/courses/courses.component.html [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+  <div class="courses__options">
+    <div class="courses__search">
+      <input
+        type="text"
+        class="input--search"
```
Comment (nastabulak) [inline src/app/components/courses/courses.component.html]: courses__input--search

File: src/app/components/courses/courses.component.scss [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+      width: 100%;
+
+      .item__header {
+        @include flex-content-direction(space-between, row);
+        align-items: flex-start;
+        padding: 5px;
+        width: 100%;
+        font-size: x-large;
+      }
+      .item__header--main {
```
Comment (nastabulak) [inline src/app/components/courses/courses.component.scss]: is main modificator?

File: src/app/components/drag-drop/drag-drop.component.html [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+    #allList="cdkDropList"
+    [cdkDropListData]="authorsOutputList"
+    [cdkDropListConnectedTo]="[confirmList]"
+    class="authors__list"
+    (cdkDropListDropped)="drop($event)">
+    <div class="authors__box" *ngFor="let item of authorsOutputList" cdkDrag>{{item.name}}</div>
+  </div>
+</div>
+
+<div class="container">
```
Comment (nastabulak) [inline src/app/components/drag-drop/drag-drop.component.html]: container?

File: src/app/components/add-course/add-course.component.html (hunk 1 above)
Comment (IrinaSendetskaya) [reply src/app/components/add-course/add-course.component.html]: changed

File: src/app/components/add-course/add-course.component.html (hunk 2 above)
Comment (IrinaSendetskaya) [reply src/app/components/add-course/add-course.component.html]: changed

File: src/app/components/add-course/add-course.component.html (hunk 3 above)
Comment (IrinaSendetskaya) [reply src/app/components/add-course/add-course.component.html]: changed

File: src/app/components/courses/courses.component.html (hunk 4 above)
Comment (IrinaSendetskaya) [reply src/app/components/courses/courses.component.html]: changed

File: src/app/components/courses/courses.component.html (hunk 5 above)
Comment (IrinaSendetskaya) [reply src/app/components/courses/courses.component.html]: changed

File: src/app/components/courses/courses.component.scss (hunk 6 above)
Comment (IrinaSendetskaya) [reply src/app/components/courses/courses.component.scss]: changed

File: src/app/components/drag-drop/drag-drop.component.html (hunk 7 above)
Comment (IrinaSendetskaya) [reply src/app/components/drag-drop/drag-drop.component.html]: changed

**Pull request merged**